scripts/quarantine_integration.yaml

```diff
@@ -1426,3 +1426,7 @@
   platforms:
     - thingy53/nrf5340/cpuapp/ns
   comment: "Configurations excluded to limit resources usage in integration builds"
+
+- platforms:
```

Comment:
btw. Do we need to update quarantines (apart from sdk-zephyr)? The `native_posix` board seems removed in scope of the same PR.

Reply:
`native_posix` will dropping after Zephyr v.4.2 (on sdk-nrf when it will be upmerged). Until then, `native_posix` might be run on CI (or another CI plans). This entry avoid running these tests.